app/kuma-dp/pkg/dataplane/envoy/remote_bootstrap.go

```diff
@@ -83,7 +83,7 @@ func (b *remoteBootstrap) Generate(ctx context.Context, url string, cfg kuma_dp.
 		case DpNotFoundErr:
 			log.Info("Dataplane entity is not yet found in the Control Plane. If you are running on Kubernetes, CP is most likely still in the process of converting Pod to Dataplane. If it takes too long, check kuma-cp logs. Retrying.", "backoff", cfg.ControlPlane.Retry.Backoff)
 		default:
-			log.Info("could not fetch bootstrap configuration. Retrying.", "backoff", cfg.ControlPlane.Retry.Backoff, "err", err.Error())
+			log.Info("Could not fetch bootstrap configuration. Make sure you are not trying to connect to global-cp. Retrying (this could help only if you're connecting to zone-cp).", "backoff", cfg.ControlPlane.Retry.Backoff, "err", err.Error())
```

Comment:
logs should be lower-case https://github.com/kumahq/kuma/issues/879#issuecomment-714472120

Reply:
`this could help only if you're connecting to zone-cp` or standalone

Reply:
Really need to introduce golang-ci for this :)